### DISCUSSION TITLE
Can't enroll in student program

### DISCUSSION THREAD

**Discussion by coolacpc** [community/community#54725] [GitHub Education]
### Hi everyone,

I'm trying to apply for the student program, but everytime i try to upload my certificate of enrollment, i keeping getting errors.
ps: i live in a different country, and i'm doing my degree online.

Has anyone had this problem before?

_No response_

**Comment by ryanlua** [comment]
Can you specify the error and region you are in?

**Reply by coolacpc** [reply]
I live in the middle east, and i'm Currently enrolled with IUBH.
I downloaded my enrollment documents from the platform, and i already have my email set to my unis email, and it was automatically recognized by github when i was trying to fill out the application.

i tried uploading the documents, both in english and german, directly from my pc camera and smartphone, but the error i keep getting is this,
![Github](https://user-images.githubusercontent.com/13906653/236706044-0a2e8a12-35ab-48ed-b452-ef513bfe9ed9.png)

**Reply by ryanlua** [reply]
The error lists what you need to do to get accepted. 

Just follow what the error asks and retry when you have fixed it.

**Reply by coolacpc** [reply]
1) The image contains my school name + logo
2) Distant Learning is mentioned
3) Profile is complete

**Reply by ryanlua** [reply]
I have no clue as to why your application was rejected then. I would just contact [GitHub Support](https://support.github.com/) as they will probably understand.

**Reply by coolacpc** [reply]
Contacted them, no answer

**Reply by ryanlua** [reply]
I would wait a while before abandoning GitHub Support, in the meantime there is an article for [Why wasn't my application to Global Campus for students approved?]( https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved) which can provide you more insight.

Additionally you can read this post of getting some of the student benefits without an ID at [How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/).

**Reply by coolacpc** [reply]
Thanks @RyanLua for this info, but i have obtained the entire documents a student can from the admissions office.
I have checked the mentioned articles above before i apply. but still, no response.

**Reply by ryanlua** [reply]
I don't think I can do anything from this point on other than just general advice. I know currently that the process for applying outside the US is a bit unreliable. 

What documents are you uploading specifically? I recommend using an unofficial transcript (like me) in which you can more than likely be able to obtain from your admissions office.

**Comment by samus-aran** [comment]
Hi @coolacpc 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

Let us know how you get on 👍
